models/Element/Note.php

```diff
@@ -156,6 +157,7 @@ public function save()
 
         if (!$isUpdate) {
             \Pimcore::getEventDispatcher()->dispatch(new ElementEvent($this), ElementEvents::POST_ADD);
+            \Pimcore::getEventDispatcher()->dispatch(new ElementEvent($this), NoteEvents::POST_ADD);
```

Comment:
Should we add a check here if there's a listener for the deprecated event and issue a deprecation message? 
E.g. 
```php
if(!empty(\Pimcore::getEventDispatcher()->getListeners(ElementEvents::POST_ADD)) { ...
```

Reply:
Sure, done -> https://github.com/pimcore/pimcore/pull/13835/commits/9ac080c19d180305e7254cf743301c12f8c04adf